test/unit/specs/components/ActionModal/components/TableInvoice.spec.js

```diff
@@ -6,25 +6,31 @@ describe(`TableInvoice`, () => {
   const localVue = createLocalVue()
   localVue.directive(`tooltip`, () => {})
 
-  const $store = {
-    getters: {
-      bondDenom: `STAKE`
-    }
-  }
-
   beforeEach(() => {
     wrapper = shallowMount(TableInvoice, {
       localVue,
-      mocks: { $store },
       propsData: {
-        amount: 1,
+        amount: 17.2,
         gasEstimate: 1234567,
-        gasPrice: 2.5e-8
+        gasPrice: 2.5e-8,
+        bondDenom: `STAKE`
       }
     })
   })
 
-  it(`should show the transaction invoice summary`, async () => {
+  it(`should render correctly`, async () => {
     expect(wrapper.vm.$el).toMatchSnapshot()
   })
+
+  it(`should display the correct subtotal`, async () => {
+    expect(wrapper.text()).toMatch(/17.2[0]+ STAKE/)
```

Comment:
try to test `wrapper.vm.subTotal` instead of `wrapper.text`

Reply:
same for the two tests below

Reply:
@colw Do you want to update them our should I?

Reply:
ah ok - fabo filled me in on the debate.

not worth bike shedding over - but my feeling is that we should test the way things are rendered with snapshots and we should use unit tests to test computed properties.

you guys sort it out so we can get it merged!

Reply:
Snapshots are only useful to let you know something changed. It has not bearing on wether it is correct or not.

If you test the internal implementation, it also means you need to change the tests, which kind of defeats the purpose of the tests.

This way, you can change the internal methods as much as you want. Change a computed property name, no worries. Change a tag order, no worries. Accidentally change that `*` into a `+`? Test fails. It's job done.

Reply:
Just to clarify my comment now that I read it with a different perspective. By "them" I meant the tests. @jbibla Sorry for any confusion